client/src/components/annotators/ImageAnnotator.vue

```diff
@@ -165,12 +166,15 @@ export default defineComponent({
           local.pendingImgs.delete(imgInternal);
         }
       });
+      let result = true;
```

Comment:
I'm going to rename this to `imageLoadSucceeded` in my branch if that's O.K.

I won't know what this does in a week :P